File: .build-tsan263/arm64-apple-macosx/debug/KSCrashRecordingTests.build/DerivedSources/resource_bundle_accessor.m
#import <Foundation/Foundation.h>

NSBundle* KSCrashRecordingTests_SWIFTPM_MODULE_BUNDLE() {
    NSURL *bundleURL = [[[NSBundle mainBundle] bundleURL] URLByAppendingPathComponent:@"KSCrash_KSCrashRecordingTests.bundle"];

    NSBundle *preferredBundle = [NSBundle bundleWithURL:bundleURL];
    if (preferredBundle == nil) {
      return [NSBundle bundleWithPath:@"/private/tmp/claude-501/-Users-alex-Documents-Code-Github-KSCrash/970ce29a-b378-4e24-bd72-2d2a41382f5a/scratchpad/pr855/.build-tsan263/arm64-apple-macosx/debug/KSCrash_KSCrashRecordingTests.bundle"];
    }

    return preferredBundle;
}

File: .build-tsan263/arm64-apple-macosx/debug/KSCrashProfiler.build/DerivedSources/resource_bundle_accessor.swift
import Foundation

extension Foundation.Bundle {
    static let module: Bundle = {
        let mainPath = Bundle.main.bundleURL.appendingPathComponent("KSCrash_KSCrashProfiler.bundle").path
        let buildPath = "/private/tmp/claude-501/-Users-alex-Documents-Code-Github-KSCrash/970ce29a-b378-4e24-bd72-2d2a41382f5a/scratchpad/pr855/.build-tsan263/arm64-apple-macosx/debug/KSCrash_KSCrashProfiler.bundle"

        let preferredBundle = Bundle(path: mainPath)

        guard let bundle = preferredBundle ?? Bundle(path: buildPath) else {
            // Users can write a function called fatalError themselves, we should be resilient against that.
            Swift.fatalError("could not load resource bundle: from \(mainPath) or \(buildPath)")
        }

        return bundle
    }()
}